### DISCUSSION TITLE
Release 0.3.0 published?

### DISCUSSION THREAD

**Discussion by aa-matthias** [blackcowmoo/Grafana-Google-Analytics-DataSource#117] [General]
Hey, thanks for the work of maintaining this plugin!

Is there any info on when Release 0.3.0 will be available in the Grafana Plugin Repo?

0.2.3 cannot be installed from official repos because it's not officially compatible with Grafana 11.
Will 0.3.0 add compatibility?

**Comment by lcc3108** [comment]
It's currently in progress.
Grafana asked for some modifications.

I couldn't process it because I was on my summer vacation, but I'm planning to proceed this week.

**Reply by aa-matthias** [reply]
thanks! hope you've had a great time!
